src/vs/workbench/services/workspaces/common/workspaceTrust.ts

```diff
@@ -887,4 +887,4 @@ class WorkspaceTrustMemento {
 	}
 }
 
-registerSingleton(IWorkspaceTrustRequestService, WorkspaceTrustRequestService, false);
+registerSingleton(IWorkspaceTrustRequestService, WorkspaceTrustRequestService, true);
```

Comment:
fyi - I have changed the signature yesterday to only support `false | InstantiationType` so I believe this won't compile

Reply:
🤦🏻‍♂️ thanks!